packages/valory/skills/termination_abci/skill.yaml

```diff
@@ -120,11 +118,7 @@ models:
       round_timeout_seconds: 30.0
       service_id: termination_abci
       service_registry_address: '0x48b6af7B12C71f09e2fC8aF4855De4Ff54e775cA'
-      setup:
```

Comment:
Same, don't understand why we remove it in some

Reply:
https://github.com/valory-xyz/open-autonomy/pull/1805#discussion_r1105375171